pylero.spec

```diff
@@ -2,21 +2,30 @@
 %define version 0.0.2
 %define unmangled_version 0.0.2
 %define release 1
+%define _unpackaged_files_terminate_build 0
 
 Summary: Python SDK for Polarion
 Name: %{name}
 Version: %{version}
 Release: %{release}
-Source0: https://github.com/RedHatQE/pylero/archive/refs/tags/%{unmangled_version}.tar.gz
 License: MIT
 Group: Development/Libraries
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 Prefix: %{_prefix}
-BuildArch: noarch
 Vendor: pylero Developers <gsun@redhat.com>
-Url: https://github.com/RedHatQE/pylero
 
-%description
+URL: https://github.com/RedHatQE/pylero
+SOURCE: %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+
+Requires: python3-%{name} == %{version}-%{release}
+Requires: python3-suds python3-click
+
+BuildArch: noarch
+BuildRequires: python3-devel
```

Comment:
- I believe this is only required for `pylero` entrypoint and since we aren't supporting it in the pkg no need to pull this big dependency

Reply:
BuildRequires is only for build the package which requires python3-devel for python package build.

The `Requires` specify what packages are dependency for the package rpm.